packages/editor/src/components/post-excerpt/check.js

```diff
@@ -1,10 +1,12 @@
 /**
  * Internal dependencies
  */
-import PostTypeSupportCheck from '../post-type-support-check';
+import { usePostTypeSupportCheck } from '../post-type-support-check';
 
-function PostExcerptCheck( props ) {
-	return <PostTypeSupportCheck { ...props } supportKeys="excerpt" />;
+export default function PostExcerptCheck( children ) {
+	return usePostExcerptCheck() ? children : null;
```

Comment:
What do you think about using `usePostTypeSupportCheck( 'excerpt' )` directly? One less public API to maintainn for us.

P.S. I don't think I've seen hooks used in ternaries before 😄

Reply:
Yeah, I added it for consistency, but I suppose they're both useless. I'll remove `usePostExcerptCheck` and deprecate `PostExcerptCheck`.

Reply:
I think we can keep `PostExcerptCheck` and just use `usePostTypeSupportCheck` internally. What do you think?